decidim-core/app/views/decidim/devise/shared/_login_boxes.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="login__box-container">
-  <%= link_to(decidim.new_user_session_path, class: "login__box") do %>
+  <div class="login__box" data-dialog-open="loginModal">
```

Comment:
Now that you've changed this to a regular `<div>`, this is no longer focusable using keyboard tab navigation.

This should be a `<button type="button">` in case it's not linking anywhere but has functionality attached to it.